# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../audacity/wx_24.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
use wxgtk2-2.4-config, wx-config is part of wx25
- Sebastian Czech <t2_@arcor.de>
--- audacity-src-1.2.3_orig/configure.in	2005-03-31 23:00:44.453246368 +0200
+++ audacity-src-1.2.3/configure.in	2005-03-31 23:07:55.473721296 +0200
@@ -516,7 +516,7 @@
 dnl --- check for required libraries ---
 
 dnl wxWindows -- we assume that if wx-config is found, wxWindows is successfully installed.
-AC_PATH_PROG(WX_CONFIG, wx-config, no, $PATH:/usr/local/bin )
+AC_PATH_PROG(WX_CONFIG, wxgtk2-2.4-config, no, $PATH:/usr/local/bin )
 if [[ "$WX_CONFIG" = "no" ]] ; then
 	AC_MSG_ERROR("Could not find wx-config: is wxWindows installed? is wx-config in your path?")
 fi
